geode-core/src/test/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImplJUnitTest.java

```diff
@@ -392,7 +388,7 @@ public void testLocatorUpdateIntervalZero() throws Exception {
 
   private void startFakeLocator() throws UnknownHostException, IOException, InterruptedException {
     server = new TcpServer(port, InetAddress.getLocalHost(), null, null, handler, new FakeHelper(),
-        Thread.currentThread().getThreadGroup(), "Tcp Server");
+        Thread.currentThread().getThreadGroup(), "Tcp Server", null);
```

Comment:
We added another parameter "protobuf handler" for TcpServer constructor to process protobuf messages. And it is nothing to do with existing code base and is only relevant for Protobuf requests. That's why we are passing null for old code.